src/geometry/manifold/basic_smooth_bundle.lean

```diff
@@ -0,0 +1,566 @@
+  (coord_change j 𝕜 ((i.1.symm.trans j.1).to_fun x)) (coord_change i j x v) = coord_change i 𝕜 x v)
+(coord_change_smooth : ∀i j : atlas H M,
+  times_cont_diff_on 𝕜 ⊤ (λp : E × F, coord_change i j (I.inv_fun p.1) p.2)
+  (set.prod (I.to_fun '' (i.1.symm.trans j.1).source) (univ : set F)))
+
+namespace basic_smooth_bundle_core
+
+variables {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
+{E : Type u} [normed_group E] [normed_space 𝕜 E]
+{H : Type*} [topological_space H] {I : model_with_corners 𝕜 E H}
+{M : Type*} [topological_space M] [manifold H M] [smooth_manifold_with_corners I M]
+{F : Type u} [normed_group F] [normed_space 𝕜 F]
+(Z : basic_smooth_bundle_core I M F)
+
+/-- Fiber bundle core associated to a basic smooth bundle core -/
+def to_topological_fiber_bundle_core : topological_fiber_bundle_core (atlas H M) M F :=
+{ base_set := λi, i.1.source,
+  is_open_base_set := λi, i.1.open_source,
+  index_at := λx, ⟨chart_at H x, chart_mem_atlas H x⟩,
+  mem_base_set_at := λx, mem_chart_source H x,
+  coord_change := λi j x v, Z.coord_change i j (i.1.to_fun x) v,
+  coord_change_self := λi x hx v, Z.coord_change_self i (i.1.to_fun x) (i.1.map_source hx) v,
+  coord_change_comp := λi j 𝕜 x ⟨⟨hx1, hx2⟩, hx3⟩ v, begin
+    have := Z.coord_change_comp i j 𝕜 (i.1.to_fun x) _ v,
+    convert this using 2,
+    { simp [hx1] },
+    { simp [local_equiv.trans_source, hx1, hx2, hx3, i.1.map_source, j.1.map_source] }
+  end,
+  coord_change_continuous := λi j, begin
+    have A : continuous_on (λp : E × F, Z.coord_change i j (I.inv_fun p.1) p.2)
+      (set.prod (I.to_fun '' (i.1.symm.trans j.1).source) (univ : set F)) :=
+      (Z.coord_change_smooth i j).continuous_on,
+    have B : continuous_on (λx : M, I.to_fun (i.1.to_fun x)) i.1.source :=
+      I.continuous_to_fun.comp_continuous_on i.1.continuous_to_fun,
+    have C : continuous_on (λp : M × F, (⟨I.to_fun (i.1.to_fun p.1), p.2⟩ : E × F))
+             (set.prod i.1.source univ),
+    { apply continuous_on.prod _ continuous_snd.continuous_on,
+      apply continuous_on.comp B continuous_fst.continuous_on (prod_subset_preimage_fst _ _) },
+    have C' : continuous_on (λp : M × F, (⟨I.to_fun (i.1.to_fun p.1), p.2⟩ : E × F))
+              (set.prod (i.1.source ∩ j.1.source) univ) :=
+      continuous_on.mono C (prod_mono (inter_subset_left _ _) (subset.refl _)),
+    have D : set.prod (i.1.source ∩ j.1.source) univ ⊆ (λ (p : M × F),
+      (I.to_fun (i.1.to_fun p.1), p.2)) ⁻¹' (set.prod (I.to_fun '' (i.1.symm.trans j.1).source) univ),
+    { rintros ⟨x, v⟩ hx,
+      simp at hx,
+      simp only [prod_mk_mem_set_prod_eq, and_true, mem_univ, mem_preimage,
+                 local_homeomorph.trans_to_local_equiv, local_homeomorph.symm_to_local_equiv ],
+      apply mem_image_of_mem,
+      simp [local_equiv.trans_source, i.1.map_source hx.1, hx.1, hx.2] },
+    convert continuous_on.comp A C' D,
+    ext p,
+    simp
+  end }
+
+@[simp] lemma base_set (i : atlas H M) :
```

Comment:
There are many things in `to_topological_fiber_bundle_core`, and I only want one of them to have the `[simp]` attribute. I don't think `[simps]` was designed for this kind of situation, but I will be happy to hear about it if it can do it!

Reply:
No, it's just for all the fields at once.